### PR TITLE
Update containers for local modules

### LINKED ISSUE #99: colabfold - No module named 'openmm'
### Description of the bug

Error with colabfold

### Command used and terminal output

```console
nextflow run proteinfold-dev --input sample_alpha.csv --outdir colabfold_test_multi --use_gpu true --mode colabfold --colabfold_server webserver --num_recycle 3 --colabfold_model_preset alphafold2_multimer_v3 -profile ifb_core

Input file 'sample_alpha.csv' contains two entries
sequence,fasta
prot1,/shared/prot1.fasta
prot2,/shared/prot2.fasta
__________

Traceback (most recent call last):
    File "/localcolabfold/colabfold-conda/bin/colabfold_batch", line 8, in <module>
      sys.exit(main())
    File "/localcolabfold/colabfold-conda/lib/python3.9/site-packages/colabfold/batch.py", line 1771, in main
      run(
    File "/localcolabfold/colabfold-conda/lib/python3.9/site-packages/colabfold/batch.py", line 1449, in run
      results = predict_structure(
    File "/localcolabfold/colabfold-conda/lib/python3.9/site-packages/colabfold/batch.py", line 529, in predict_structure
      pdb_lines = relax_me(pdb_lines=unrelaxed_pdb_lines[key], use_gpu=use_gpu_relax)
    File "/localcolabfold/colabfold-conda/lib/python3.9/site-packages/colabfold/batch.py", line 300, in relax_me
      from alphafold.relax import relax
    File "/localcolabfold/colabfold-conda/lib/python3.9/site-packages/alphafold/relax/relax.py", line 18, in <module>
      from alphafold.relax import amber_minimize
    File "/localcolabfold/colabfold-conda/lib/python3.9/site-packages/alphafold/relax/amber_minimize.py", line 25, in <module>
      from alphafold.relax import cleanup
    File "/localcolabfold/colabfold-conda/lib/python3.9/site-packages/alphafold/relax/cleanup.py", line 23, in <module>
      from openmm import app
  ModuleNotFoundError: No module named 'openmm'
```

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by athbaltzis**
This PR consists of updates to the AlphaFold2 and ColabFold containers so as to include dependencies updates, such as openmm 7.7.0 and python3.10, for improving the performance.

Closes #99 

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/proteinfold/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/proteinfold _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/local/colabfold_batch.nf [hunk 1]
```diff
@@ -3,7 +3,7 @@ process COLABFOLD_BATCH {
     label 'process_medium'
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'docker://quay.io/nf-core/proteinfold_colabfold:1.1.0' :
```
Comment (JoseEspinosa) [inline modules/local/colabfold_batch.nf:6]: I think we need the `quay.io` in the URL, don't we?

Comment (JoseEspinosa) [reply modules/local/colabfold_batch.nf:6]: I mean the container is not in dockerhub anymore, should it be?

File: modules/local/mmseqs_colabfoldsearch.nf [hunk 2]
```diff
@@ -3,10 +3,9 @@ process MMSEQS_COLABFOLDSEARCH {
     label 'process_high_memory'
 
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'docker://quay.io/nf-core/proteinfold_colabfold:1.1.0' :
```
Comment (JoseEspinosa) [inline modules/local/mmseqs_colabfoldsearch.nf:6]: Same here and in all the other instances

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Have a question about image URL when using singularity

File: modules/local/colabfold_batch.nf (hunk 1 above)
Comment (l-mansouri) [reply modules/local/colabfold_batch.nf:6]: I had the same question!

Comment (athbaltzis) [reply modules/local/colabfold_batch.nf:6]: It's not necessary, since the default docker.registry is quay.io

**Review by l-mansouri — COMMENTED** [review]
Look good to me!!

**Review by JoseEspinosa — APPROVED** [review]
Awesome! :rocket:

**Pull request merged**